packages/ripple-binary-codec/src/enums/definitions.json

```diff
@@ -2695,14 +2777,15 @@
     "temUNKNOWN": -264,
     "temSEQ_AND_TICKET": -263,
     "temBAD_NFTOKEN_TRANSFER_FEE": -262,
-    "temBAD_AMM_TOKENS": -261,
-    "temXCHAIN_EQUAL_DOOR_ACCOUNTS": -260,
-    "temXCHAIN_BAD_PROOF": -259,
-    "temXCHAIN_BRIDGE_BAD_ISSUES": -258,
-    "temXCHAIN_BRIDGE_NONDOOR_OWNER": -257,
-    "temXCHAIN_BRIDGE_BAD_MIN_ACCOUNT_CREATE_AMOUNT": -256,
-    "temXCHAIN_BRIDGE_BAD_REWARD_AMOUNT": -255,
-    "temEMPTY_DID": -254,
+    "temBAD_MPTOKEN_TRANSFER_FEE": -261,
+    "temBAD_AMM_TOKENS": -260,
+    "temXCHAIN_EQUAL_DOOR_ACCOUNTS": -259,
+    "temXCHAIN_BAD_PROOF": -258,
+    "temXCHAIN_BRIDGE_BAD_ISSUES": -257,
+    "temXCHAIN_BRIDGE_NONDOOR_OWNER": -256,
+    "temXCHAIN_BRIDGE_BAD_MIN_ACCOUNT_CREATE_AMOUNT": -255,
+    "temXCHAIN_BRIDGE_BAD_REWARD_AMOUNT": -254,
+    "temEMPTY_DID": -253,
```

Comment:
This changes all the numbers, which isn't backwards-compatible.

Reply:
Auto-generating _is_ sufficient, I'm saying this is a rippled problem.

Reply:
This is only a problem if you plan to merge this PR in as-is - if it's just for a beta version or something, it doesn't really matter.